README.md

```diff
@@ -88,9 +88,12 @@ Here's a handy boilerplate for starting a new file in [this gist](https://gist.g
 Plan-specific tweaks can be managed using the following global functions:
 | Helper function |  Return value  |
 |---|---|
-| `wc_calypso_bridge_has_ecommerce_features()`   | This will return `true` for all ecommerce-related plans *(such as the Ecommerce and Ecommerce Trial.)* |
-| `wc_calypso_bridge_is_ecommerce_plan()`   | Returns `true` if site has the specific Ecommerce plan. e.g., `false` for the trial plan. |
+| `wc_calypso_bridge_has_ecommerce_features()`   | This will return `true` for all ecommerce-related plans *(including all dotCom, Woo Express plans, and the Ecommerce Free Trial)* |
+| `wc_calypso_bridge_is_ecommerce_plan()`   | Returns `true` if the site has a paid Ecommerce plan (including all dotCom and Woo Express plans) |
```

Comment:
It would be better if we use WordPress.com instead of `dotCom`.
```suggestion
| `wc_calypso_bridge_has_ecommerce_features()`   | This will return `true` for all ecommerce-related plans *(including all WordPress.com, Woo Express plans, and the Ecommerce Free Trial)* |
| `wc_calypso_bridge_is_ecommerce_plan()`   | Returns `true` if the site has a paid Ecommerce plan (including all WordPress.com and Woo Express plans) |
```